playbooks/rfcs.md

```diff
@@ -20,12 +20,12 @@ process with as much bang for buck as possible given our size.
 
 ### TLDR: Our process looks like:
 
-- Someone opens an issue on a repo that begins with "RFC: ", and that causes a message to be posted to Slack's #dev
-  room via Peril.
+- Someone opens a pull request on a repo (or an issue, if it's a repo that doesn't have an rfc directory to hold RFC docs) that begins with "RFC: "; this causes a message to be posted to Slack's #dev
```

Comment:
I'm a little confused here so want to check my understanding:
- We can create RFCs from any repository, defaulting to an `/rfc` directory
- The rfc directory on the main branch is effectively a list of passed RFCs

Should engineers therefore view the rfcs directory as a directory of policies and decisions we've made, or should a given PR include changes elsewhere as well?

Reply:
Your understanding is correct, but as far as I understand it, you're asking whether an RFC PR should also include any actions that may be taken to resolve the PR. Is that right?

So in this PR, I've included actions that are required to implement the RFC, i.e.
- create the `/rfc` dir
- update the playbook

This makes sense to me in this case because everything affected by this RFC is in the same repo. So the PR contains:
1. the case I made (the RFC itself)
2. the decision made by the org to accept the RFC (the comments)
3. the changes required to implement the RFC (the two actions listed above)

I could definitely see a case for separating (3) out into a separate PR, and I'm happy to do that if you think it makes more sense. OTOH it feels nicely atomic to have it all bundled in one place 🤷 

What do you think @erikdstock ? Which approach is preferable?

Reply:
To clarify: by "which approach" I mean:

- include the changes required to implement the RFC inside the PR, along with the RFC itself, or
- restrict the PR to only include the RFC, create a new PR for the changes required to implement it

Reply:
@erikdstock any thoughts?

Reply:
I think my confusion was around:
a) now RFCs can happen on any repo? _Yes, and peril looks for PRs to a document in the `/rfc` folder or with a title starting with `RFC: `_
b) whether an RFC is accepted or not, what happens to the PR? _it should be merged at resolution with the merge documenting the result_

If these are correct that addresses it for me - I think it would be nice to have some markdown frontmatter showing whether an RFC passed or not.

Reply:
ping @erikdstock, when you have a moment

Reply:
✅ Looks good, thanks!